tests/precompile/precompile_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// e2e implements the e2e tests.
```

Comment:
```suggestion
// e2e implements the e2e tests.
```
```suggestion
// e2e tests for precompiles.
```

Reply:
Removing this comment since we don't do this anywhere else